code/controllers/subsystem/timer.dm

```diff
@@ -331,15 +332,20 @@ SUBSYSTEM_DEF(timer)
 
 
 /datum/controller/subsystem/timer/Recover()
-	//Find the current timer sub-subsystem in global and recover its buckets etc
+	// Find the current timer sub-subsystem in global and recover its buckets etc
 	var/datum/controller/subsystem/timer/timerSS = null
 	for(var/global_var in global.vars)
 		if (istype(global.vars[global_var],src.type))
 			timerSS = global.vars[global_var]
-	second_queue |= timerSS.second_queue
-	hashes |= timerSS.hashes
-	timer_id_dict |= timerSS.timer_id_dict
-	bucket_list |= timerSS.bucket_list
```

Comment:
There's no point doing `|=` when the lists of the new timer SS are empty anyways.

Reply:
|= is the equivalent of doing a list.Copy()

What you're now doing is passing the reference to the lists from timerSS to the new one.

Is assigning the references to timerSS's lists that what you're intending to happen instead of copying over the contents?

Reply:
@Timberpoes Yes. `reset_buckets()` makes new lists anyways.